module06/en.subject.tex

```diff
@@ -554,7 +554,7 @@ \section*{Examples}
 # Example 1.3:
 lr2.loss_(y, y_hat)
 # Output:
-80.83996294128525
+161.67992588257047
```

Comment:
You cannot do that because student will implement the definition of the loss from the previous module where the division by 2 is in the formula. Plus as we suggest to look to the video of the MOOC, within the video the definition also display a division by 2